components/ADempiere/Form/VAllocation/components/Payments.vue

```diff
@@ -0,0 +1,277 @@
+          </div>
+        </el-card>
+        <el-card
+          class="box-card"
+          style="width: 50%;float: right;"
+        >
+          <div>
+            <el-form
+              :inline="true"
+              label-position="left"
+              style="padding: 10px !important;"
+            >
+              <el-col :span="24">
+                <el-form-item
+                  label="Pagado:"
+                >
+                  <b>
+                    350,63
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Asignado:"
+                >
+                  <b>
+                    170,00
+                  </b>
+                </el-form-item>
+              </el-col>
+              <el-col :span="24">
+                <el-form-item
+                  label="Monto Abierto:"
+                >
+                  <b>
+                    10,00
+                  </b>
+                </el-form-item>
+              </el-col>
+            </el-form>
+          </div>
+        </el-card>
+      </div>
+    </el-card>
+  </div>
+</template>
+
+<script>
+import { defineComponent, ref } from '@vue/composition-api'
+// import lang from '@/lang'
+// components and mixins
+import Carousel from '@theme/components/ADempiere/Carousel'
+// import formMixin from '@theme/components/ADempiere/Form/formMixin.js'
+// import fieldsList from './fieldList.js'
+// import SearchCriteria from './components/SearchCriteria/'
+// import Invoices from './components/Invoices/index'
+// import Receipt from './components/Receipt/index'
+
+// Utils and Helper Methods
+// import { isEmptyValue } from '@/utils/ADempiere/valueUtils'
+// import { showMessage } from '@/utils/ADempiere/notification'
+
+// API Request Methods
+// import {
+//   listPaymentSelection,
+//   paymentSelection
+// } from '@/api/ADempiere/form/VPayPrint.js'
+
+export default defineComponent({
+  name: 'Payments',
+
+  components: {
+    Carousel
+  },
+
+  props: {
+    metadata: {
+      type: Object,
+      default: () => {
+        return {}
+      }
+    }
+  },
+
+  setup(props, { root }) {
+    /**
+     * Refs
+     */
+
+    const tableData = ref([])
+
+    const headers = ref([
+      {
+        label: 'ID',
+        columnName: '',
+        width: '50'
+      },
+      {
+        label: 'Numero',
+        columnName: '',
+        width: '130'
+      },
+      {
+        label: 'Fecha',
+        columnName: '',
+        width: '150'
+      },
+      {
+        label: 'Total',
+        columnName: '',
+        width: '150'
+      },
+      {
+        label: 'Convertido',
+        columnName: '',
+        width: '150'
+      },
+      {
+        label: 'Pagado',
+        columnName: '',
+        width: '150'
+      },
+      {
+        label: 'Monto Asignado',
+        columnName: '',
+        width: '150'
+      },
+      {
+        label: 'Monto Abierto',
```

Comment:
@elsiosanchez please add translation.

Reply:
Added the translation in the following commit [68211fe](https://github.com/solop-develop/frontend-default-theme/pull/442/commits/68211fe355873ffe49725eb888013312491e1af4)

Reply:
Added the translation in the following commit [68211fe](https://github.com/solop-develop/frontend-default-theme/pull/442/commits/68211fe355873ffe49725eb888013312491e1af4)